src/loaders/zarrLoader.js

```diff
@@ -89,9 +124,12 @@ export default class ZarrLoader {
     return [data];
   }
 
-  async getRaster({ z, level }) {
+  async getRaster({ z }) {
     const source = this.isPyramid ? this._data[z] : this._data;
-    const { data } = await source.getRaw([level, null, null]);
+    const selection = [...this.chunkIndex];
+    selection[this.xIndex] = null;
+    selection[this.yIndex] = null;
```

Comment:
Can you add a comment/link about why this is necessary?

Reply:
Related to the above. Using a selection rather than chunk key allows us to get the entire image pane regardless of whether it is tiled or chunked. `null` is equivalent here to using `[:]` in numpy.

in python:

```python
z = zarr.open('my_data.zarr')
z.shape
# [4, 5000, 5000]
z.chunks
# [1, 500, 500]
arr = z[1, :, :]
arr.shape
# [1, 5000, 5000]
```

in javascript:

```javascript
z = await openArray({ store: 'my_data.zarr' });
console.log(z.shape);
# [4, 5000, 5000]
console.log(z.chunks);
# [1, 500, 500]
arr = z.getRaw([1, null, null]);
console.log(arr.shape)
# [1, 5000, 5000]
```